FinancialTracker/Firebase/AuthManager.swift

```diff
@@ -132,8 +135,9 @@ class AuthManager {
         }
     }
     
-    func addTransactionToCurrentUser(_ amount: Double, category: Category, completionHandler: @escaping (FirebaseError?, Bool) -> Void) {
-        databaseManager.addTransactionToCurrentUser(amount, category: category) { firebaseError, success in
+    func addTransactionToUserByUID(_ amount: Double, category: Category, completionHandler: @escaping (FirebaseError?, Bool) -> Void) {
```

Comment:
The name should be the way it was `ToCurrentUser.`
Reason - this is authManager caller for this function and when its called from this class it is not needed to have byUID, it is by UID only in the database class because there i am sending it to different users.

Reply:
Name was is a bit misleading. That is why you need proper comments.